exercises/diffie-hellman/src/lib.rs

```diff
@@ -1,11 +1,11 @@
-pub fn private_key(_p: u64) -> u64 {
-    unimplemented!()
+pub fn private_key(p: u64) -> u64 {
+    unimplemented!("Pick a private key greater than and less than {}", p)
 }
 
-pub fn public_key(_p: u64, _g: u64, _a: u64) -> u64 {
-    unimplemented!()
+pub fn public_key(p: u64, g: u64, a: u64) -> u64 {
+    unimplemented!("Calculate public key using prime numbers {} and {}, and private key {}", p, g, a)
 }
 
-pub fn secret(_p: u64, _b_pub: u64, _a: u64) -> u64 {
-    unimplemented!()
-}
+pub fn secret(p: u64, b_pub: u64, a: u64) -> u64 {
+    unimplemented!("Calculate secret key using prime number {}, public key {}, and private key {}", p, b_pub, a)
+}
```

Comment:
there was a trailing newline in this file. Since this PR should be focused on dealing with underscore-prefixed names and not with whitespace, the trailing newline should not be removed in this PR

(also the trailing newline should stay anyway, by convention)

(Same comment for all other files)